escuela/src/components/nav/NavigationBar.js

```diff
@@ -27,6 +27,11 @@ export default () => {
               Home
 						</NavLink>
           </li>
+          <li className="nav-item" >
+            <NavLink to="/Assignment" className="nav-link" >
```

Comment:
URLs should be lowercased, `<Navlink to="/assignment" />`

Reply:
fixed